charts/prometheus-redis-exporter/templates/deployment.yaml

```
@@ -28,6 +28,10 @@ spec:
         - name: {{ . }}
       {{- end}}
       {{- end }}
+      {{- with .Values.specSecurityContext }}
```

```suggestion
      {{- with .Values.podSecurityContext }}
```

Security context is commonly referred to in the repository's charts by `securityContext`, `containerSecurityContext` and `podSecurityContext`. Since users and contributors are familiar with these fields, `podSecurityContext` would fit, would it not? 

Please, enter the new field in `values.yaml` with its default value.